faq/billing.mdx

```diff
@@ -122,4 +123,56 @@ You can add a billing contact directly from the User Account section of your Sca
 
 ## How can I obtain a SEPA Mandate?
 
-Refer to the [How to add a payment method](/console/billing/how-to/add-payment-method#how-to-add-a-sepa-mandate) documentation page for a complete guide on how to add a SEPA mandate.
\ No newline at end of file
+Refer to the [How to add a payment method](/console/billing/how-to/add-payment-method#how-to-add-a-sepa-mandate) documentation page for a complete guide on how to add a SEPA mandate.
+
+## My payment verification code expired. What should I do?
+
+If your verification code expires, please [contact our support team](https://console.scaleway.com/support/tickets/create) to request a new magic code.
+
+## I tried to add a credit card but received the message 'Card already registered'
+
+You can only use one credit card per Organization. If you receive this message, make sure you do not have the same card already registered to another Scaleway account.
+
+If you only have one payment method and need to use it on more than one account, [contact our support team](https://console.scaleway.com/support/tickets/create) to request an individual verification.
+
+## I've added a physical credit card, but it was declined. Why?
+
+If your credit card complies with the payment requirements but still was declined, please [contact our support team](https://console.scaleway.com/support/tickets/create) for assistance.
+
+## I added a credit card but didn’t receive a magic code. What do I do?
+
+It usually takes up to 24 hours to receive a magic code.
+
+If after that period you still don't receive it, contact your bank or credit card provider to request the transaction code.
+
+## How long does it take to receive a magic code?
+
+Usually, the magic code (transaction code) shows up in your bank statement in up to 24 hours.
+
+## Why can’t I delete my credit card?
+
+It might not be possible to remove your credit card from your account if you have one or more outstanding invoices. If that is the case, make sure to pay the open invoice(s) and retry.
```

Comment:
```suggestion
It might not be possible to remove your credit card from your account if you have one or more outstanding invoices. If that is the case, make sure you have paid any issued invoice(s) and have no pending consuming over the current calendar month, and retry.
```